### PR TITLE
[FIX/UPDATE] Setting permissions and commands

### PR DESCRIPTION
**Pull request by vinnyfs89**
Changing `lendingbot.service` permission and setting the command `sudo systemctl daemon-reload` before enable `lendingbot.service`

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] **I have read CONTRIBUTING.md**
- [ ] **I fully understand [Github Flow.](https://guides.github.com/introduction/flow/)**
- [ ] **My code adheres to the [code style of this project.](https://poloniexlendingbot.readthedocs.io/en/latest/contributing.html)**
- [ ] **I have updated the documentation in /docs if I have changed the config, arguments, logic in how the bot works, or anything that understandably needs a documentation change.**
- [ ] **I have updated the config file accordingly if my change requires a new configuration setting or changes an existing one.**
- [ ] **I have tested the bot with no issues for 24 continuous hours. If issues were experienced, they have been patched and tested again.**

### CODE REVIEW

**Comment by utdrmac** [comment]
Are you sure these changes are not distro/OS specific? I didn't have to chmod 644 or do a daemon-reload on my OS to get things to work. My distro is ReadyNAS/FreeBSD. On which distro did you test this?

**Comment by vinnyfs89** [comment]
In my case I had to follow these steps to work correctly using a raspberry pi.

The operating system I used for the test was the [RASPBIAN STRETCH LITE](https://www.raspberrypi.org/downloads/raspbian/)

Worked perfectly!

**Comment by utdrmac** [comment]
Could you test your changes on other OSes? Your "fix" may break others.

**Comment by vinnyfs89** [comment]
What if the text is specific for raspberry pi users who use Raspbian? I find it impractical to test all possible operating systems ...

**Comment by rnevet** [comment]
If it's a specific setup solution then I suggest adding it in the documentation.

File: docs/installation.rst [hunk 1]
```diff
@@ -66,10 +66,25 @@ To download the bot you can either:
         WantedBy=multi-user.target
 
     Credit to GitHub user utdrmac.
+    
+    
+    The permission on the unit file needs to be set to 644 :
```
Comment (Evanito) [inline docs/installation.rst]: Since this is only necessary on some OSs, please rephrase this to say "may" need to be set...
Very useful Troubleshooting tip! :)
